src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeHttpWrapperTests.cs

```diff
@@ -126,48 +125,15 @@ public async Task PostAsync_HasContent_SendsRequestWithSerializedContent()
         }
 
         [TestMethod]
-        public async Task PostAsync_FailsToExecuteRequest_ExceptionCaughtAndLogged()
+        public async Task PostAsync_Exception_ExceptionNotCaught()
         {
             var httpMessageHandler = new FakeHttpMessageHandler(message =>
-                throw new NotImplementedException("this is a test"));
-            var logger = new TestLogger();
-            var testSubject = CreateTestSubject(httpMessageHandler: httpMessageHandler, logger: logger);
-
-            var response = await testSubject.PostAsync(DefaultUri, null, CancellationToken.None);
-            response.Should().BeNull();
-            logger.AssertPartialOutputStringExists("this is a test");
-        }
-
-        [TestMethod]
-        public async Task PostAsync_AggregateException_ExceptionCaughtAndLogged()
-        {
-            var httpMessageHandler = new FakeHttpMessageHandler(message =>
-                throw new AggregateException(
-                    new List<Exception>
-                    {
-                        new ArgumentNullException("this is a test1"),
-                        new NotImplementedException("this is a test2")
-                    }));
-
-            var logger = new TestLogger();
-            var testSubject = CreateTestSubject(httpMessageHandler, logger: logger);
-
-            var response = await testSubject.PostAsync(DefaultUri, null, CancellationToken.None);
-            response.Should().BeNull();
-            logger.AssertPartialOutputStringExists("this is a test1");
-            logger.AssertPartialOutputStringExists("this is a test2");
-        }
-
-        [TestMethod]
-        public async Task PostAsync_CriticalException_ExceptionNotCaught()
-        {
-            var httpMessageHandler = new FakeHttpMessageHandler(message =>
-                throw new StackOverflowException());
+                throw new NotImplementedException());
```

Comment:
I removed all the exception handling tests and changed the exception to a non critical one (as the critical ones were already thrown beforehand).